tests/e2e/logging/test_openshift-logging.py

```diff
@@ -237,7 +239,54 @@ def test_respin_osd_pods_to_verify_logging(self, create_pvc_and_deploymentconfig
         project2_filecount = elasticsearch_pod_obj.exec_cmd_on_pod(
             command=f'es_util --query=project.{project2}.*/_count', out_yaml_format=True
         )
-        logger.info(f'The files in the project2 {project2_filecount}')
-
         assert project2_filecount['_shards']['successful'] == 0, f"No files found in project {project2}"
-        logger.info(f'The files in the project 2 {project2_filecount}')
+        logger.info(f'Total number files in project 2 {project2_filecount}')
+
+    @pytest.mark.polarion_id("OCS-665")
+    @tier4
```

Comment:
Please change to workloads tier

Reply:
@ebenahar are we segregating workload tier again into tier1 and tier2 etc...